docs/changes.rst

```diff
@@ -22,6 +22,16 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.14.0 - 2017-07-20
```

Comment:
Note that this is in UTC, with one hour leeway - I'd bump it now so it doesn't break the build later.